### PR TITLE
Bump Substrate & runtime version

### PR DESCRIPTION
**Pull request by gavofyork**

### CODE REVIEW

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -584,7 +555,7 @@ construct_runtime! {
 
 		// Consensus support.
 		Authorship: authorship::{Module, Call, Storage},
-		Staking: staking::{default, OfflineWorker},
```
Comment (gavofyork) [inline runtime/kusama/src/lib.rs]: @bkchr `OfflineWorker` is no longer supported?

Comment (bkchr) [reply runtime/kusama/src/lib.rs]: That is okay. This was never a valid syntax. I'm not even sure what `OfflineWorker` should refer too. If that should refer to offchain workers, they are implemented like `OnInitialize` etc.

File: runtime/kusama/src/lib.rs [hunk 2]
```diff
@@ -610,13 +581,11 @@ construct_runtime! {
 		Slots: slots::{Module, Call, Storage, Event<T>},
 		Registrar: registrar::{Module, Call, Storage, Event, Config<T>},
 
-		// Simple nicknames module.
-		// KUSAMA: Remove before mainnet
-		Nicks: nicks::{Module, Call, Storage, Event<T>},
```
Comment (bkchr) [inline runtime/kusama/src/lib.rs]: Why is nicks removed?

File: runtime/kusama/src/lib.rs [hunk 3]
```diff
@@ -584,7 +555,7 @@ construct_runtime! {
 
 		// Consensus support.
 		Authorship: authorship::{Module, Call, Storage},
-		Staking: staking::{default, OfflineWorker},
+		Staking: staking::{default},
```
Comment (bkchr) [inline runtime/kusama/src/lib.rs]:
```suggestion
		Staking: staking,
```

File: runtime/kusama/src/lib.rs (hunk 2 above)
Comment (gavofyork) [reply runtime/kusama/src/lib.rs]: Was scheduled to be removed (identity replaces it)

**Pull request merged**